packages/main/src/Carousel.js

```diff
@@ -152,6 +152,14 @@ const metadata = {
 		_itemWidth: {
 			type: Integer,
 		},
+
+		/**
+		 * If set to true navigation arrows are hidden
+		 * @private
+		 */
+		_hideNavigationArrows: {
```

Comment:
What is the difference between this property and the existing property hideNavigation?

Reply:
hideNavigation hides the whole navigation including the pagination

Reply:
Ok. Use `noAttribute: true` on this property which will prevent the rendering of a `_hide-navigation-arrows` attribute to the element. You don't need to pollute the DOM with such internal stuff.